components/molecules/Friend/FriendGroup.tsx

```diff
@@ -88,6 +89,7 @@ const FriendGroup: React.FC<FriendProps> = (props) => {
         onPress,
         notificationType,
         messageSeen,
+        showNames = true
```

Comment:
Let's add this default value within the props definition above instead of here, since I want us to remove all these const things